### PR TITLE
Update "Report a problem with a translation" page

### PR DESCRIPTION
**Pull request by kulsoomzahra**
Addressing the issue: https://gitlab.torproject.org/tpo/web/community/-/issues/158

### CODE REVIEW

File: content/localization/translation-problem/contents.lr [hunk 1]
```diff
@@ -19,7 +19,7 @@ body:
...
 
 * If you are already a [Tor translator](../becoming-tor-translator), you can simply find the string and fix it in [transifex](https://www.transifex.com/otf/torproject/).
-* If you don't know how to find the string to fix, you can [open a ticket on our Bugtracker](https://support.torproject.org/misc/bug-or-feedback/), under the **Community/Translations** component.
-  Before opening a ticket, see the currently [open tickets for Translations](https://trac.torproject.org/projects/tor/query?status=accepted&status=assigned&status=merge_ready&status=needs_information&status=needs_review&status=needs_revision&status=new&status=reopened&component=Community%2FTranslations&order=priority) because maybe it is already reported.
+* If you don't know how to find the string to fix, you can [open a ticket on our Bugtracker](https://support.torproject.org/misc/bug-or-feedback/), under the **Community/Localization** repository.
+  Before opening a ticket, see the currently [open tickets for Translations](https://gitlab.torproject.org/groups/tpo/-/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name%5B%5D=Community) because maybe it is already reported.
```
Comment (gusgustavo) [inline content/localization/translation-problem/contents.lr:23]: The ticket link is wrong. To see all open tickets: 
https://gitlab.torproject.org/groups/tpo/-/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name[]=Localization 
and 
https://gitlab.torproject.org/tpo/community/l10n/-/issues

**Review by gusgustavo — CHANGES_REQUESTED** [review]
Please read my comment, then git squash your commits and I will review.

**Comment by kulsoomzahra** [comment]
Sure, let me know if any other changes are to be made. Thanks

**Comment by gusgustavo** [comment]
@emmapeel2 can you review this PR? Thanks!

**Comment by emmapeel2** [comment]
Apologies, I fixed the issue the other day, without realising there was a pull request. 

This pull request can be closed.

Thanks!

**Pull request closed without merging**